airbyte-integrations/connectors/source-asana/source_asana/manifest.yaml

```diff
@@ -0,0 +1,2566 @@
+    $ref: "#/definitions/base_stream"
+    name: "portfolios_compact"
+    primary_key: "gid"
+    schema_loader:
+      type: InlineSchemaLoader
+      schema:
+        $ref: "#/definitions/portfolios_compact_schema"
+    $parameters:
+      path: "/portfolios"
+      extractor_path: "data"
+    retriever:
+      $ref: "#/definitions/retriever"
+      requester:
+        $ref: "#/definitions/requester"
+        request_parameters:
+          owner: "{{ stream_slice['users_gid'] }}"
+          workspace: "{{ stream_slice['workspace_gid'] }}"
+      partition_router:
+        - type: SubstreamPartitionRouter
+          parent_stream_configs:
+            - stream: "#/definitions/workspaces_stream"
+              parent_key: "gid"
+              partition_field: "workspace_gid"
+        - type: SubstreamPartitionRouter
+          parent_stream_configs:
+            - stream: "#/definitions/users_stream"
+              parent_key: "gid"
+              partition_field: "users_gid"
+
+  portfolios_compact_partition_router:
+    type: SubstreamPartitionRouter
+    parent_stream_configs:
+      - stream: "#/definitions/portfolios_compact_stream"
+        parent_key: "gid"
+        partition_field: "portfolios_gid"
+
+  portfolios_stream:
+    $ref: "#/definitions/base_stream"
+    name: "portfolios"
+    primary_key: "gid"
+    schema_loader:
+      type: InlineSchemaLoader
+      schema:
+        $ref: "#/definitions/portfolios_schema"
+    $parameters:
+      path: "/portfolios/{{ stream_slice['portfolios_gid'] }}"
+      extractor_path: "data"
+    retriever:
+      $ref: "#/definitions/retriever"
+      paginator:
+        $ref: "#/definitions/paginator_base"
+      partition_router:
+        $ref: "#/definitions/portfolios_compact_partition_router"
+
+  custom_fields_stream:
+    $ref: "#/definitions/base_stream"
+    name: "custom_fields"
+    primary_key: "gid"
+    schema_loader:
+      type: InlineSchemaLoader
+      schema:
+        $ref: "#/definitions/custom_fields_schema"
+    $parameters:
+      path: "/workspaces/{{ stream_slice['workspace_gid'] }}/custom_fields"
+      extractor_path: "data"
+    retriever:
+      $ref: "#/definitions/retriever"
+      partition_router:
+        $ref: "#/definitions/workspaces_partition_router"
+
+  organization_exports_stream:
+    $ref: "#/definitions/base_stream"
+    name: "organization_exports"
+    primary_key: "gid"
+    schema_loader:
+      type: InlineSchemaLoader
+      schema:
+        $ref: "#/definitions/organization_exports_schema"
+    $parameters:
+      path: "/organization_exports/{{ stream_slice.gid }}"
+      extractor_path: "data"
+    retriever:
+      $ref: "#/definitions/retriever"
+      partition_router:
+        - type: ListPartitionRouter
+          values: "{{ config['organization_export_ids' ] }}"
+          cursor_field: gid
+
+  events_stream:
+    $ref: "#/definitions/base_stream"
+    name: "events"
+    primary_key: "created_at"
+    schema_loader:
```

Comment:
Getting the following extra fields when running `discover` command:

- `path`
- `extractor_path`
- `$parameters`

Missing field `type`